docs/user-guide/node-selection/index.md

```diff
@@ -198,3 +198,203 @@ must be satisfied for the pod to schedule onto a node.
 
 For more information on inter-pod affinity/anti-affinity, see the design doc
 [here](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/design/podaffinity.md).
+
+## Taints and tolerations (beta feature)
+
+### Specifying taints and tolerations
+
+Node affinity, described earlier, allows a pod to "be attracted to" a set of nodes
+(either as a preference or a hard requirement). Taints are the opposite -- they allow
+a *node* to *repel* a set of pods.
+
+More concretely: *taints* go on nodes, and *tolerations* go on pods. A taint on a node
+means "repel all pods that do not tolerate the taint."
+
+You add a taint to a node using [kubectl taint](https://kubernetes.io/docs/user-guide/kubectl/kubectl_taint/).
+For example,
+
+```shell
+kubectl taint nodes node1 key=value:NoSchedule
+```
+
+places a taint on node `node1`. The taint has key `key`, value `value`, and taint effect `NoSchedule`.
+This means that no pod will be able to schedule on `node1` unless it has a matching toleration.
+The toleration is specified in the pod template. Both of the following tolerations "match" the
+taint created by the `kubectl taint` line above, and thus a pod with either toleration would be able
+to schedule onto `node1`:
+
+```yaml
+tolerations: 
+   - key: "key"
+     operator: "Equal"
+     value: "value"
+     effect: "NoSchedule"
+```
+
+```yaml
+tolerations: 
+   - key: "key"
+     operator: "Exists"
+     effect: "NoSchedule"
+```
+
+A toleration "matches" a taint if the `key`s are the same and the `effect`s are the same, and
```

Comment:
The problem is that if you do that, then the reader hasn't seen "operator' anywhere yet. By putting an example toleration first, they at least have some idea what operator looks like.